src/instruments.ts

```diff
@@ -411,6 +411,10 @@ export class InstrumentRepository {
             {'symbol': 'APEUSDT', 'tickSize': 0.001},
             {'symbol': 'BNXUSDT', 'tickSize': 0.01}
         ])
+
+        this._lookup.set('Kraken',[
+            {'symbol': 'BTC/USD', 'tickSize': 1}
```

Comment:
I have a jupyter notebook that I can use to pull all of the symbols and tick sizes so can do that once it's merged 👍